src/headless/tasks/inasafe_wrapper.py

```diff
@@ -31,8 +31,8 @@
 from safe.gui.widgets.dock import set_provenance_to_project_variables
 
 # Initialize qgis_app
-from safe.test.utilities import get_qgis_app  # noqa
-QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+from safe.test.qgis_app import qgis_app  # noqa
+APP, IFACE = qgis_app()
```

Comment:
Use the new one

Reply:
It's still not perfect. For instance you get a 'mocked' object and not the real original class type. So it depends